src/adapter/src/coord/dataflows.rs

```diff
@@ -145,11 +145,34 @@ impl Coordinator {
         dataflow: DataflowDesc,
         instance: ComputeInstanceId,
     ) -> DataflowDescription<Plan> {
+        #[allow(deprecated)]
         self.ship_dataflow(dataflow, instance)
             .await
             .expect("failed to ship dataflow")
     }
 
+    /// Broadcasts a finalized dataflow to all workers.
+    pub(crate) async fn ship_dataflow1(
```

Comment:
Here the `1` variant is the new function. But in other places things that are the `1` variant are the old function. Make this consistent: I recommend using `X_deprecated` for old function names, or otherwise `X_new` for new functions. `1` is not clear.